test/integration/server-discovery-and-monitoring/topology_description.test.ts

```diff
@@ -14,7 +14,22 @@ describe('TopologyDescription (integration tests)', function () {
     await client.close();
   });
 
+  beforeEach(async function () {
+    client = this.configuration.newClient();
```

Comment:
What's the rationale behind adding this new beforeEach block without a `afterEach client.close`?

Reply:
The `afterEach` was already there (lines 13-15)